sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -137,7 +137,7 @@ The following section provides several code snippets illustrating common pattern
 * [Manage Custom Models Synchronously](#manage-custom-models-synchronously)
 
 ### Recognize Content
-Recognize text and table data, along with their bounding box coordinates, from documents.
+Recognize text, table, and and selection marks like radio buttons and check boxes data, along with their bounding box coordinates, from documents.
```

Comment:
Since `selection markS` is pluralized, it would make more sense to say `texts, tables, ...` (or make everything singular).
The `like radio buttons and check boxes` makes it hard to parse the sentence.

`...text, table and selection mark data` sounds better to me, but I understand you're trying to be clear.